src/app/shared/services/user-session.service.ts

```diff
@@ -61,7 +61,10 @@ export class UserSessionService implements OnDestroy {
 
   refresh(): Observable<void> {
     const refresh$ = this.currentUserService.refresh().pipe(shareReplay(1));
-    refresh$.subscribe(() => this.userProfileUpdated$.next());
+    refresh$.subscribe({
+      next: () => this.userProfileUpdated$.next(),
+      error: () => {},
```

Comment:
Can you add a comment indicating why it's necessary to have an empty function here?